.circleci/config.yml

```diff
@@ -1096,69 +1090,61 @@ workflows:
           skip-when-no-change: true
 
       - test-driver:
-          name: be-tests-bigquery-ee
+          matrix:
+            parameters:
+              driver: ["bigquery", "bigquery-cloud-sdk", "googleanalytics", "sqlite"]
+          name: be-tests-<< matrix.driver >>
```

Comment:
Any reason these shouldn't be named `-ee` anymore? `be-tests-sqlite` instead of `be-tests-sqlite-ee`